Cargo.toml

```diff
@@ -17,21 +17,21 @@
 
 [workspace]
 members = [
-    "datafusion",
-    "datafusion-common",
-    "datafusion-expr",
-    "datafusion-jit",
-    "datafusion-physical-expr",
-    "datafusion-cli",
+    "datafusion/rust/core",
+    "datafusion/rust/common",
+    "datafusion/rust/expr",
+    "datafusion/rust/jit",
+    "datafusion/rust/physical-expr",
+    "datafusion/rust/cli",
+    "datafusion/rust/proto",
```

Comment:
the rust directory here seems a bit redundant to me, do you expect us to have other language implementations here?

Reply:
Agree